Formula/vulkan-tools.rb

```diff
@@ -0,0 +1,98 @@
+class VulkanTools < Formula
+  desc "Vulkan utilities and tools"
+  homepage "https://github.com/KhronosGroup/Vulkan-Tools"
+  url "https://github.com/KhronosGroup/Vulkan-Tools/archive/refs/tags/v1.3.248.tar.gz"
+  sha256 "9f06f13e588034e323df9222532b76518ace8bf5e4eb4ca9fbc6960addf03a93"
+  license "Apache-2.0"
+  head "https://github.com/KhronosGroup/Vulkan-Tools.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on "python@3.11" => :build
+  depends_on "glslang"
+  depends_on "vulkan-headers"
+  depends_on "vulkan-loader"
+
+  on_macos do
+    depends_on "molten-vk"
+  end
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "libx11"
+    depends_on "libxcb"
+    depends_on "libxkbfile"
+    depends_on "libxrandr"
+    depends_on "wayland"
+    depends_on "wayland-protocols"
+  end
+
+  def install
+    if OS.mac?
+      # account for using already-built MoltenVK instead of the source repo
+      inreplace "mac_common.cmake",
+                "${MOLTENVK_DIR}/MoltenVK/icd/MoltenVK_icd.json",
+                "${MOLTENVK_DIR}/share/vulkan/icd.d/MoltenVK_icd.json"
+      inreplace buildpath.glob("*/macOS/*/*.cmake") do |s|
+        s.gsub! "${MOLTENVK_DIR}/MoltenVK/include",
+                "${MOLTENVK_DIR}/include"
+        s.gsub! "${MOLTENVK_DIR}/MoltenVK/dylib/macOS/libMoltenVK.dylib",
+                "${MOLTENVK_DIR}/lib/libMoltenVK.dylib"
+      end
+    end
+
+    args = [
+      "-DBUILD_ICD=ON",
+      "-DBUILD_CUBE=ON",
+      "-DBUILD_VULKANINFO=ON",
+      "-DINSTALL_ICD=OFF", # we will manually place it in a nonconflicting location
+      "-DGLSLANG_INSTALL_DIR=#{Formula["glslang"].prefix}",
+      "-DVULKAN_HEADERS_INSTALL_DIR=#{Formula["vulkan-headers"].prefix}",
+      "-DVULKAN_LOADER_INSTALL_DIR=#{Formula["vulkan-loader"].prefix}",
+    ]
+    args += if OS.mac?
+      ["-DMOLTENVK_REPO_ROOT=#{Formula["molten-vk"].prefix}"]
+    else
+      [
+        "-DBUILD_WSI_DIRECTFB_SUPPORT=OFF",
+        "-DBUILD_WSI_WAYLAND_SUPPORT=ON",
+        "-DBUILD_WSI_XCB_SUPPORT=ON",
+        "-DBUILD_WSI_XLIB_SUPPORT=ON",
+      ]
+    end
+    system "cmake", "-S", ".", "-B", "build", *args, *std_cmake_args
+    system "cmake", "--build", "build", "--target", "VulkanTools_generated_source"
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+
+    (lib/"mock_icd").install (buildpath/"build/icd/VkICD_mock_icd.json").realpath,
+                             shared_library("build/icd/libVkICD_mock_icd")
+
+    return unless OS.mac?
+
+    targets = [
+      Formula["molten-vk"].opt_lib/shared_library("libMoltenVK"),
+      Formula["vulkan-loader"].opt_lib/shared_library("libvulkan.#{Formula["vulkan-loader"].version}"),
```

Comment:
```suggestion
      Formula["vulkan-loader"].opt_lib/shared_library("libvulkan", Formula["vulkan-loader"].version),
```
Might need to do this to make it compatible with Linux; not sure. Using a glob would be simpler.